cli/config.toml

```diff
@@ -0,0 +1,30 @@
+title = "CLI config file"
```

Comment:
Do we need this title? 
- Title goes away if configuration change via command `mainflux-cli  config key value`
- Title is not used
So please consider for removing title